src/main/java/com/algorand/algosdk/crypto/LogicsigSignature.java

```diff
@@ -61,6 +61,10 @@ public LogicsigSignature(
         this.msig = msig;
     }
 
+    public LogicsigSignature(byte[] logic) {
```

Comment:
Also, the definition of the class LogicsigSignature does not have a description about what this class is. 
It will be very helpful to follow with the logic in these classes if the LogicSignature class is described.